test/cloudtest/test_computed.py

```diff
@@ -38,9 +41,19 @@ def test_computed_sizing(mz: MaterializeApplication) -> None:
     mz.environmentd.sql("DROP CLUSTER sized1 CASCADE")
 
 
-def test_computed_shutdown(mz: MaterializeApplication) -> None:
+@pytest.mark.parametrize("failpoint", [False, True])
+def test_computed_shutdown(mz: MaterializeApplication, failpoint: bool) -> None:
     """Test that dropping a cluster or replica causes the associated computeds to shut down."""
 
+    if failpoint:
+        mz.set_environmentd_failpoints("after_catalog_drop_replica=panic")
+
+    def sql_ignore_crash(sql: str) -> None:
+        try:
+            mz.environmentd.sql(sql)
+        except InterfaceError:
+            pass
```

Comment:
Nit: Print error with a non-error level to make debugging easier.

Reply:
Generally agree, but as it is used now, the function will create an InterfaceError all the time. Maybe I can rename the function to `crashing_sql` or so and assert that an error happens.